src/components/Layout/styles/Layout.module.css

```
@@ -0,0 +1,41 @@
+.container {
+    display: flex;
+    flex-direction: column;
+}
+
+.overflowContainer {
+    overflow-x: hidden;
+    overflow-y: auto;
+    flex: 1;
+    height: auto;
+    min-height: 78px;
+    max-height: 20vh;
```

I think instead of a `min-height` you could declare a `flex-basis` of `78px` and then set `flex-shrink` to 0, to prevent the item to shrink beyond its base size.
I also think that `height` is redundant here. So probably I'd change this part to the following (using the [`flex` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/flex)):
```suggestion
    flex-basis: 1 0 78px;
    max-height: 20vh;
```